resources/views/layouts/admin.blade.php

```diff
@@ -17,6 +17,7 @@
     @vite(['resources/sass/app.scss', 'resources/js/app.js'])
 
     <link rel="stylesheet" href="{{ asset('css/nav-menu.css') }}">
+    <link rel="stylesheet" href="{{ asset('css/confirmation.css') }}">
 
     <script src="https://kit.fontawesome.com/dbc5b98639.js" crossorigin="anonymous"></script>
```

Comment:
Download this kit because it may cause some blocking in other browsers

Reply:
I'm sorry, I'm not sure what I am suppose to download

Reply:
@Ayaka0001  - Go to this link https://kit.fontawesome.com/dbc5b98639.js copy and paste all of the code in the screen. Save it in assets folder as .js file

Reply:
is this right?

 inside layouts/admin.blade.php 
  <link rel="stylesheet" href="{{ asset('css/nav-menu.css') }}">
    <link rel="stylesheet" href="{{ asset('css/confirmation.css') }}">
    <link rel="stylesheet" href="{{ asset('js/.js') }}">
</head>

Reply:
<link rel="stylesheet" href="{{ asset('css/nav-menu.css') }}">
    <link rel="stylesheet" href="{{ asset('css/confirmation.css') }}">
    <link rel="stylesheet" href="{{ asset('js/.js') }}">
</head>

Reply:
<img width="602" alt="Screenshot 2024-10-31 at 21 22 26" src="https://github.com/user-attachments/assets/86a0b775-8be3-4dc3-9c44-78edd8d56737">

Reply:
If this is correct, the fontawsome icons are not showing in my pages.